docs/other-models.md

```diff
@@ -101,6 +101,33 @@ And confirm they were logged correctly with:
 llm logs -n 1
 ```
 
+### Azure OpenAI Service
+
+LLM supports OpenAI models deployed via [Azure OpenAI Service](https://azure.microsoft.com/en-us/products/ai-services/openai-service). These can also be added using the same configuration mechanism.
+
+The `model_id` is the name LLM will use for the model. The `model_name` is the deployment name of the model on Azure OpenAI Service. 
+
+The `api_base` will be the Azure OpenAI Service endpoint. The `api_key_name` is the name of the Azure OpenAI Service API key stored using the {ref}`api-keys` feature.
+
+You must add `api_version` to use a specific version of the Azure OpenAI Service API.
+
+Set `api_type` to `azure` to specify that the OpenAI client library should use the Azure OpenAI Service API.
```

Comment:
Worth noting that "azure_ad" is not supported, as far as I can tell (keyless auth with AAD).

Reply:
```suggestion
Set `api_type` to `azure` to specify that the OpenAI client library should use the Azure OpenAI Service API. Note that the type of `azure_ad` (keyless auth) is not yet supported.
```